LibTCPServer.emx

```diff
@@ -876,7 +876,8 @@ if (config.logToStdOut)
 		bool firstPort = true;
 		for (int i = actorData->numPorts; --i >= 0; ++portDescriptor ) {
 			RTProtocol* port = RTObjectOf( RTProtocol, portDescriptor->offset, this );
-			if (port->getName() == "external" && port->getInfo()->protocol->name == "External")
+			if (!RTMemoryUtil::strcmp(port->getName(), "external") &&
```

Comment:
Same question as above

Reply:
Here the warning is understandable since we compare two char pointers instead of strings. So this change looks correct.